HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthUtils.swift

```diff
@@ -11,7 +11,7 @@ import UIKit
  
  */
 public func giniHealthBundle() -> Bundle {
-    Bundle.module
+    Bundle.resource
```

Comment:
Clarification: Why we need this change? 
We have down here the `giniHealthBundleResource()` function that's providing the resources.